content/posts/the-three-rs-rust-repl-and-runners/index.mdx

```diff
@@ -0,0 +1,234 @@
+---
+title: 'The Three "R"s: Rust, REPL, Runner'
+date: 2021-01-22
+---
+
+REPLs and runners can be an easier and quick way to iterate and execute code. Apply the same Rust example with a Cargo package and different Rust REPLS & Runners to see the differences between each tool.
```

Comment:
This reads a bit awkwardly to me. Perhaps:

> REPLs and runners can be a quick and easy way to iterate and execute code.

I'm not quite following the second sentence so not sure what to suggest there.

Reply:
Your suggestion reads much better to me as well. I added a better segue to a slightly altered version of the second sentence.
```
--- a/content/posts/the-three-rs-rust-repl-and-runners/index.mdx
+++ b/content/posts/the-three-rs-rust-repl-and-runners/index.mdx
@@ -3,7 +3,7 @@ title: 'The Three "R"s: Rust, REPL, Runner'
 date: 2021-01-22
 ---
 
-REPLs and runners can be an easier and quick way to iterate and execute code. Apply the same Rust example with a Cargo package and different Rust REPLS & Runners to see the differences between each tool.
+REPLs and runners can be a quick and easy way to iterate and execute code. Let's try different Rust REPLs & runners to see the differences between each tool.
 
 ## Intro
 tl;dr [Jump to the code](#cargo-project)
```